deploy/crds/imagemonitor_v1alpha1_imagemonitor_cr.yaml

```diff
@@ -3,5 +3,4 @@ kind: ImageMonitor
 metadata:
   name: example-imagemonitor
 spec:
-  # Add fields here
-  namespaces: 3
+  excludePattern: ""
```

Comment:
This actually stops anything working, currently it looks like you need to set an excludePattern to something or it excludes everything. .i.e `excludePattern: "thismatchesnothing"`. Removing it entirely also results in an empty one being added, which has the same result.

Reply:
This is fixed in the other PR